email_app/app/safenet_comm/network.js

```diff
@@ -1,19 +1,24 @@
-import { shell } from 'electron';
+import { shell, remote } from 'electron';
 import { initializeApp, fromAuthURI } from '@maidsafe/safe-node-app';
 import { getAuthData, saveAuthData, clearAuthData,
   parseUrl, showError } from '../utils/app_utils';
 import pkg from '../../package.json';
 import { CONSTANTS } from '../constants';
 import 'babel-polyfill';
 
+const isDevMode = process.execPath.match(/[\\/]electron/);
+const app = remote.app;
 
+const cwd = process.cwd();
+const isWindows = process.platform === 'win32' ? '.cmd' : '';
```

Comment:
I think the variable name is misleading as it sounds like it holds a boolean value. Perhaps something like `electronExt` could be better, or make it have the full electron executable name `electron.cmd` and name it `electronCmd`?